Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -529,6 +529,7 @@ public async Task RunStartupTasksAsync(CancellationToken cancellationToken)
 
         private IEnumerable<Task> StartEntryPoints(IEnumerable<IServerEntryPoint> entryPoints, bool isBeforeStartup)
```

Comment:
Should the return type be changed to `IReadOnlyList`?

Reply:
Looking at the source for `WhenAll`, it looks like it tests for array, then ICollection; so I don't think changing the return type would have any real-world benefit.

If you want I can still change it for internal use